sklearn/ensemble/tests/test_iforest.py

```diff
@@ -0,0 +1,139 @@
+
+"""
+Testing for Isolation Forest algorithm (sklearn.ensemble.iforest).
+"""
+
+# Authors: Nicolas Goix <nicolas.goix@telecom-paristech.fr>
+#          Alexandre Gramfort <alexandre.gramfort@telecom-paristech.fr>
+# License: BSD 3 clause
+
+import numpy as np
+
+from sklearn.utils.testing import assert_array_equal
+from sklearn.utils.testing import assert_array_almost_equal
+from sklearn.utils.testing import assert_raises
+from sklearn.utils.testing import assert_true
+from sklearn.utils.testing import assert_warns
+
+from sklearn.grid_search import GridSearchCV, ParameterGrid
+from sklearn.ensemble import IsolationForest
+from sklearn.cross_validation import train_test_split
+from sklearn.datasets import load_boston, load_iris
+from sklearn.utils import check_random_state
+
+from scipy.sparse import csc_matrix, csr_matrix
+
+rng = check_random_state(0)
+
+# load the iris dataset
+# and randomly permute it
+iris = load_iris()
+perm = rng.permutation(iris.target.size)
+iris.data = iris.data[perm]
+iris.target = iris.target[perm]
+
+# also load the boston dataset
+# and randomly permute it
+boston = load_boston()
+perm = rng.permutation(boston.target.size)
+boston.data = boston.data[perm]
+boston.target = boston.target[perm]
+
+
+def test_iforest():
+    """Check Isolation Forest for various parameter settings."""
+    rng = check_random_state(0)
+    X_train, X_test, y_train, y_test = train_test_split(iris.data,
+                                                        iris.target,
+                                                        random_state=rng)
+    grid = ParameterGrid({"max_samples": [0.5, 1.0],
+                          "bootstrap": [True, False]})
+
+    for params in grid:
+        IsolationForest(random_state=rng,
+                        **params).fit(X_train, y_train).predict(X_test)
```

Comment:
what do you test here? That is runs? If so maybe you can do it on an even smaller dataset.